x/twap/store.go

```diff
@@ -15,11 +15,11 @@ import (
 // Two records are deleted per incentive record:
 // 1. by time index
 // 2. by pool index
-// Therefore, setting this to 1000 means 500 complete incentive records are deleted per block.
+// Therefore, setting this to 200 means 100 complete incentive records are deleted per block.
```

Comment:
Isn't this 200 complete records now that we don't 2x write it?

Reply:
Ah yeah, this logic stands for when we backport but you are right, should change it for main

Reply:
Fixed https://github.com/osmosis-labs/osmosis/pull/7638/commits/8bafac05ae5b2cf8d4159c9f86c70e77e5436ea6

Will revert in the backport